### PR TITLE
Update WikipediaSuggestion.scala

### PR DESCRIPTION
**Pull request by jo-pol**
@DANS-KNAW/easy 
proper url encoding

### CODE REVIEW

**Review by rvanheest — APPROVED** [review]
lol, yes. definitely approved!!!

File: src/main/scala/workshop4/assignments/WikipediaSuggestion.scala [hunk 1]
```diff
@@ -44,11 +45,7 @@ class WikipediaSuggestion extends Application {
   }
 
   def searchWikipedia(word: String): Observable[List[String]] = {
-    def urlifyWord(word: String): String = {
-      word.replace(" ", "%20")
-    }
-
-    val url = s"https://en.wikipedia.org/w/api.php?action=opensearch&search=${urlifyWord(word)}"
+    val url = s"https://en.wikipedia.org/w/api.php?action=opensearch&search=${URLEncoder.encode(word, "UTF-8")}"
```
Comment (janvanmansum) [inline src/main/scala/workshop4/assignments/WikipediaSuggestion.scala:48]: This works, but it actually uses HTML form encoding, which is similar, but not exactly the same as percent-encoding. I use [URI.toASCIIString()](https://docs.oracle.com/javase/8/docs/api/java/net/URI.html#toASCIIString--). This, of course, requires you to construct a `URI` object first:

```scala
new URI("https", "en.wikipedia.org", "/w/api.php", "action=opensearch&search=zoek een café", null).toASCIIString
```

**Comment by janvanmansum** [comment]
@jo-pol @rvanheest: was this tested? Travis has found an error because the wrong `URLEncoder` was apparently imported.

**Comment by janvanmansum** [comment]
@jo-pol @rvanheest : is somebody going to fix this? Otherwise I'll close this one to avoid too much clutter in our list of open PRs.

**Comment by jo-pol** [comment]
It was just to show a way without reinventing wheels

**Pull request closed without merging**